sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze.py

```diff
@@ -206,19 +206,20 @@ def test_all_successful_passing_text_document_input_entities_task(self, client):
             polling_interval=self._interval(),
         ).result()
 
-        action_results = list(response)
-
-        assert len(action_results) == 1
-        action_result = action_results[0]
-
-        assert action_result.action_type == AnalyzeActionsType.RECOGNIZE_ENTITIES
-        assert len(action_result.document_results) == len(docs)
-
-        for doc in action_result.document_results:
-            self.assertEqual(len(doc.entities), 4)
-            self.assertIsNotNone(doc.id)
-            for entity in doc.entities:
-                self.assertIsNotNone(entity.text)
+        pages = list(response)
+        assert len(pages) == len(docs)
+
+        for document_results in pages:
+            assert len(document_results) == 1
+            document_result = document_results[0]
+            assert isinstance(document_result, RecognizeEntitiesResult)
+            assert len(document_result.entities) == 4
+            assert document_result.id is not None
+            for entity in document_result.entities:
+                assert entity.text is not None
+                assert entity.category is not None
+                assert entity.offset is not None
+                assert entity.confidence_score is not None
```

Comment:
I suppose someday we should go through and convert all the unittest assert helpers to `assert` since we have a funny mix of both now. Not blocking on this PR, though